mlem/contrib/sklearn.py

```diff
@@ -106,3 +107,27 @@ def process(
                 **predict_proba_args
             )
         return mt
+
+
+class SklearnTransformer(SklearnModel):
+    valid_types: ClassVar = (
+        TransformerMixin,
+        _VectorizerMixin,
+    )
+    type: ClassVar = "sklearn_transformer"
+
+    @classmethod
+    def process(
+        cls, obj: Any, sample_data: Optional[Any] = None, **kwargs
```

Comment:
there is new argument for this function, please see updated code for other clasess